source/fileInfo.go

```diff
@@ -175,6 +175,8 @@ type GitInfo struct {
 	AuthorDate time.Time `json:"authorDate"`
 	// The commit date.
 	CommitDate time.Time `json:"commitDate"`
+	// The commit message's Body.
+	Body string `json:"Body"`
```

Comment:
Can you make it `json:"body"` (which is in line with the others)

Reply:
Done, will combine the commits and push now.